lib/charms/bootstrap.py

```diff
@@ -23,14 +25,14 @@ def bootstrap_charm_deps():
         return
     # bootstrap wheelhouse
     if os.path.exists('wheelhouse'):
-        apt_install(['python3-pip', 'python3-yaml'])
+        package_install(['python3-pip', 'python3-yaml'])
```

Comment:
I don't think this is the right approach since `apt` package names aren't always the same as `yum` package names. We'd need to split this up earlier so the package names themselves can also change based on the distro.